### ISSUE TITLE
Doesn't update

### ISSUE THREAD

**Issue by vitaly-t** [alanshaw/david#71]
Just how often does David update? I've fixed dependencies 2 hours ago, and they still come up as out of date, while David uses the old package.json.

Is there something extra I need to do or it just doesn't work?

**Comment by vitaly-t** [comment]
UPDATE: It did update, 3 hours later. That's not fast enough. Is there a way to make it happen faster?

**Comment by alanshaw** [comment]
I think it's a bug see https://github.com/alanshaw/david-www/issues/178. I haven't got time to fix this right now but would appreciate a pull request.

**Comment by seriema** [comment]
Not sure if this is the same thing, but when I run `david` it reports `All dependencies up to date`. Yet if I do `npm list` I can see `grunt-contrib-imagemin@0.9.2` but the latest is actually 0.9.4 ([published four months ago](https://github.com/gruntjs/grunt-contrib-imagemin/releases/tag/v0.9.4)).

**Comment by alanshaw** [comment]
What does the entry for `grunt-contrib-imagemin` look like in your package.json?

**Comment by seriema** [comment]
`"grunt-contrib-imagemin": "^0.9.2",`

**Comment by alanshaw** [comment]
That's up to date according to your range

**Comment by alanshaw** [comment]
That's up to date according to your range

**Comment by seriema** [comment]
Not sure I understood. On npm the latest is 0.9.4, but I can't get david to grab it when I run `david update`. It thinks that 0.9.2 is the latest, when it clearly isn't and 0.9.4 should be a valid update.

**Comment by seriema** [comment]
Not sure I understood. On npm the latest is 0.9.4, but I can't get david to grab it when I run `david update`. It thinks that 0.9.2 is the latest, when it clearly isn't and 0.9.4 should be a valid update.

**Comment by seriema** [comment]
This is still happening and either David doesn't work as I think or .. I don't know. I have `"karma": "^0.13.2",` in my `package.json`. When I run `david update` I'm expecting it to update to `0.13.8` but it doesn't.

**Comment by alanshaw** [comment]
Fair point, but as you say `npm update` updates your existing dependencies within the range of versions stated in your package.json. David's purpose has always been to identify and install dependencies who's versions are higher than the version range stated in your package.json.

**Comment by seriema** [comment]
Ah... I've missed that distinction. The David website marks any dependency out of date. Maybe that's the difference between the yellow and red badge? If it's within range or not? I thought it had to do with minor vs major version.
